tensorflow/python/eager/pywrap_tfe_src.cc

```diff
@@ -347,7 +347,14 @@ TFE_TensorHandle* TFE_Py_NumpyToTensorHandle(PyObject* obj) {
   tensorflow::Tensor t;
   auto cppstatus = tensorflow::NdarrayToTensor(obj, &t);
   if (cppstatus.ok()) {
-    return TFE_NewTensorHandle(t);
+    TF_Status* status = TF_NewStatus();
```

Comment:
Oh, wait, sorry, this doesn't need to be changed. This is invoking `TFE_NewTensorHandle(const tensorflow::Tensor& t)` and not `TFE_NewTensorHandle(const TF_Tensor* t)`.

So, I think no change is needed to this file.